allensdk/brain_observatory/filtered_running_speed/_schemas.py

```diff
@@ -0,0 +1,43 @@
+import argschema
+from argschema.fields import Nested, String
+
+
+class InputParameters(argschema.ArgSchema):
+    output_path = argschema.fields.OutputFile(
+        required=True,
+        description="Filtered running speed hdf5 output file."
+    )
+
+    mapping_pkl_path = argschema.fields.InputFile(
+        required=True,
+        help="path to pkl file containing raw stimulus information",
+    )
+    behavior_pkl_path = argschema.fields.InputFile(
+        required=True,
+        allow_none=True,
+        help="path to pkl file containing raw stimulus information",
+    )
+    replay_pkl_path = argschema.fields.InputFile(
+        required=True,
+        allow_none=True,
+        help="path to pkl file containing raw stimulus information",
+    )
+    sync_h5_path = argschema.fields.InputFile(
+        required=True,
+        help="path to h5 file containing synchronization information",
+    )
+
+    def __init__(self, *args, **kwargs):
```

Comment:
I don't think this constructor is necessary.

Reply:
I've removed this.